events/class_tests/possible_participant_tests.py

```diff
@@ -0,0 +1,78 @@
+from django.core.exceptions import ValidationError
+from ..models import PossibleMeeting, EventParticipant, PossibleParticipant, Event
+from django.utils import timezone
+from datetime import datetime
+import pytest
+
+
+ROW_DUPLICATION_ERROR = "The User is already registered in this possible meeting date"
+INVALID_PARTICIPANT_REGISTRATION = "The participant is not part of this event"
+
+
+def create_possible_participant(participant_id, possible_meeting_id):
+    return PossibleParticipant(participant_id=participant_id, possible_meeting_id=possible_meeting_id)
+
+
+@pytest.mark.django_db
+class TestPossibleParticipant():
+
+    def test_row_duplication(self):
+        event_participant = EventParticipant.objects.filter(is_creator=True).first()
+        possible_meeting = PossibleMeeting.objects.get(participant_id=event_participant.id)
+
+        with pytest.raises(ValidationError, match=ROW_DUPLICATION_ERROR):
+            create_possible_participant(event_participant, possible_meeting).save()
+            create_possible_participant(event_participant, possible_meeting).save()
+
+    def test_invalid_registeration_of_pareticipant_to_meeting(self):
+        participant_of_event1 = EventParticipant.objects.filter(is_creator=True, event_id__title='event1')[0]
```

Comment:
```suggestion
        participant_of_event1 = EventParticipant.objects.filter(is_creator=True, event_id__title='event1').first()
```

In my opinion this is a more readable way to get the same object.

Reply:
Done. Thanks! 👍